packages/server/lib/util/settings.coffee

```diff
@@ -13,6 +13,11 @@ log      = require("../log")
 
 fs = Promise.promisifyAll(fs)
 
+# TODO move default cypress.json out from here into scaffolding for example
+defaultCypressConfig = {
+  "$schema": "./cypress/cypress-schema.json"
```

Comment:
Why is this necessary? I am **very** against changing the default `cypress.json` to include a property. I didn't know what a `cypress-schema.json` was before today and I had no idea how it could or couldn't be useful.

Seeing this seeded as the default property would be confusing for the majority of our users.

Reply:
maybe it would be confusing. But maybe 99% of users do not look at `cypress.json` file at all. And the ones that try to edit this file would benefit greatly from intellisense that tells what options ARE available and what they do. 

Then other long term solution would be
- to put schema on http://schemastore.org/json/ but then it would not live close to the `cypress.json` and I don't know what to do about versioning
- editor-specific file, but then we would need to put these files out, no one is going to copy / use them